AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/AWSCognitoUserPoolTokens.swift

```diff
@@ -38,6 +39,38 @@ public struct AWSCognitoUserPoolTokens: AuthCognitoTokens {
         self.expiration = expiration
     }
 
+    public init(idToken: String,
+                accessToken: String,
+                refreshToken: String,
+                expiresIn: Int? = nil) {
```

Comment:
Do developers use this class init directly in their code? I think we should just expose `init(idToken, accessToken, refreshToken)` judging from the deprecation message

Reply:
Mmmm.. They don't use it.. But I can not be certain about it. Hence I kept it as a optional param and marked it as deprecated. 
Hopefully if some customers are using it could complain about the deprecation with a new issue, but that is highly unlikely.

Reply:
I think keeping `expiresIn` as an optional parameter means we're essentailly not deprecating it (the logic of the deprecated init just got moved over to the new init). Imagine a major version bump where we remove the deprecated init, but developers do not have a breaking change and continue to use the new init if they were passing in `expiresIn`. I may not have the full picture of how AWSCognitoUserPoolTokens is created and used, so just my thought on `expiresIn` being an optional parameter and the fact that this init supports both use cases appears to be a pretty safe, non-breaking change.

Reply:
Thought more about it.. Made the new init non-public.